draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -286,13 +315,13 @@ SD-JWT permutations, both with and without various constituent parts.
 An SD-JWT without Disclosures and without a KB-JWT:
 <Issuer-signed JWT>~
 
-An SD-JWT without Disclosures and with a KB-JWT:
-<Issuer-signed JWT>~<KB-JWT>
-
 An SD-JWT with Disclosures and without a KB-JWT:
 <Issuer-signed JWT>~<Disclosure 1>~<Disclosure N>~
 
-An SD-JWT with Disclosures and with a KB-JWT:
+An SD-JWT+KB without Disclosures:
```

Comment:
The text for these examples should be outside the ```-part. (Can't add a suggestion here, needs to be fixed in the PR.)

Reply:
FWIW that whole thing is currently enclosed in a figure (see https://www.ietf.org/archive/id/draft-ietf-oauth-selective-disclosure-jwt-08.html#section-5-11) and in the interest of moving along this PR from last year, maybe that change could be addressed in a separate PR or issue?

Reply:
created issue #428 so will "resolve" this conversation here